source/usr/lib/mcvirt/parser.py

```diff
@@ -658,6 +663,14 @@ def parse_arguments(self, script_args=None):
             vm_object.reset()
             self.print_status('Successfully reset VM')
 
+        elif action == 'clear_deadlock':
+            node = rpc.get_connection('node')
+            success = node.deadlock_escape()
```

Comment:
Probably nicer to have:
    if node.deadlock_escape():
        self.print_status(...)
    else:
        ....